app/utils/hentFraApi.ts

```diff
@@ -0,0 +1,13 @@
+import søkerMock from '~/mock/søkerMock';
+
+export const hentSøkerFornavn = () => {
+  const søker = søkerMock;
+  //her skal vi fetche fra API
+  let fornavn: string = søker.visningsnavn.trim();
+  try {
+    fornavn = fornavn.split(' ')[0];
+  } catch (e) {
+    fornavn = '!';
+  }
```

Comment:
Her burde vi ha en form for feilhåndtering. Dere kan eventuelt legge det i en egen oppgave 😊

Reply:
Har lagt til en errormelding i konsollen, men oppretter en egen oppgave for skikkelig håndtering!